src-docs/src/views/resizable_container/resizable_container_example.js

```diff
@@ -313,6 +338,30 @@ export const ResizableContainerExample = {
       demo: <ResizableContainerVertical />,
       snippet: verticalSnippet,
     },
+    {
+      source: [
+        {
+          type: GuideSectionTypes.JS,
```

Comment:
One last quick docs tweak, now that our example file is in Typescript, we need to make this a `.TSX` extension in order for the code to load with the correct file extension in CodeSandbox
```suggestion
          type: GuideSectionTypes.TSX,
```